components/tikv_util/src/log.rs

```diff
@@ -33,6 +33,25 @@ macro_rules! error {
     };
 }
 
+#[macro_export]
+macro_rules! error_unknown {
```

Comment:
Please add some comments to explain the intention of this macro

Reply:
👍 comment added.